share/completions/nmcli.fish

```diff
@@ -6,6 +6,15 @@ set -l nmcli_connection show up down add modify clone edit delete monitor reload
 set -l nmcli_device status show set connect reapply modify disconnect delete monitor wifi lldp help
 set -l nmcli_agent secret polkit all help
 
+# If the value could not be obtained, a string such as "id" is assigned to a variable.
+set -l nmcli_list_id (for i in (nmcli -g NAME connection show); string join \t $i "Connection name"; end); set -q nmcli_list_id[1]; or set -l nmcli_list_id id
```

Comment:
These variables get set only once - when the nmcli completions are loaded.
So it might be better to put these into functions